packages/block-library/src/columns/edit.js

```diff
@@ -240,29 +243,82 @@ function Placeholder( { clientId, name, setAttributes } ) {
 	const { replaceInnerBlocks } = useDispatch( blockEditorStore );
 	const blockProps = useBlockProps();
 
+	const [ isAccessibleModalOpen, setAccessibleModalOpen ] = useState( false );
+	const accessibleTitle = sprintf(
+		// translators: %s: Name of the block.
+		__( 'Get Started with %s Block' ),
+		get( blockType, [ 'title' ] )
+	);
+	const accessibleModalDescriptionId = `accessible-description-modal-${ clientId }`;
+	const ariaPropsModal = {
+		aria: {
+			describedby: accessibleModalDescriptionId,
+		},
+	};
+
 	return (
-		<div { ...blockProps }>
-			<__experimentalBlockVariationPicker
-				icon={ get( blockType, [ 'icon', 'src' ] ) }
-				label={ get( blockType, [ 'title' ] ) }
-				variations={ variations }
-				onSelect={ ( nextVariation = defaultVariation ) => {
-					if ( nextVariation.attributes ) {
-						setAttributes( nextVariation.attributes );
-					}
-					if ( nextVariation.innerBlocks ) {
-						replaceInnerBlocks(
-							clientId,
-							createBlocksFromInnerBlocksTemplate(
-								nextVariation.innerBlocks
-							),
-							true
-						);
-					}
-				} }
-				allowSkip
-			/>
-		</div>
+		<>
+			<div { ...blockProps }>
+				<Button
+					className="accessible-focus-on-block-enter"
+					onClick={ () => setAccessibleModalOpen( true ) }
+				>
+					{ accessibleTitle }
+				</Button>
+				{ isAccessibleModalOpen && (
```

Comment:
I really don't like this. I am duplicating too much code I think. Would be cool at some point to figure out how to pull this content in to the Modal via a ref or something. However, that may not be much more efficient.

Reply:
Hi, @alexstine

We could store `__experimentalBlockVariationPicker` into a variable then use it in both places. What do you think?

Example:

```js
const variationPicker = (
  <__experimentalBlockVariationPicker
	  icon={ get( blockType, [ 'icon', 'src' ] ) }
	  label={ get( blockType, [ 'title' ] ) }
	  variations={ variations }
	  onSelect={ ( nextVariation = defaultVariation ) => {
		  if ( nextVariation.attributes ) {
			  setAttributes( nextVariation.attributes );
		  }
		  if ( nextVariation.innerBlocks ) {
			  replaceInnerBlocks(
				  clientId,
				  createBlocksFromInnerBlocksTemplate(
					  nextVariation.innerBlocks
				  ),
				  true
			  );
		  }
	  } }
	  allowSkip
  />
);
```